web/src/main/webapp/xslt/common/index-utils.xsl

```diff
@@ -682,19 +682,4 @@
                           else $fieldName"/>
   </xsl:function>
 
-
-  <xsl:function name="gn-fn-index:json-escape" as="xs:string?">
-    <xsl:param name="v" as="xs:string?"/>
-    <xsl:choose>
-      <xsl:when test="normalize-space($v) = ''"></xsl:when>
-      <xsl:otherwise>
-        <xsl:value-of select="replace(replace(replace(replace(replace($v,
-                  '\\','\\\\'),
-                  $doubleQuote, $escapedDoubleQuote),
-                  '&#09;', '\\t'),
-                  '&#10;', '\\n'),
-                  '&#13;', '\\r')"/>
-      </xsl:otherwise>
-    </xsl:choose>
-  </xsl:function>
```

Comment:
Maybe we should keep it with this content for maintaining the compatibility with schemas where they keep using `gn-fn-index:json-escape` instead of the new `util:escapeForJson`:
```suggestion
  <xsl:function name="gn-fn-index:json-escape" as="xs:string?">
  <!-- This functions is deprecated. Please update your code to use util:escapeForJson function -->
  
    <xsl:param name="v" as="xs:string?"/>
    <xsl:choose>
      <xsl:when test="normalize-space($v) = ''"></xsl:when>
      <xsl:otherwise>
        <xsl:value-of select="util:escapeForJson($v)"/>
      </xsl:otherwise>
    </xsl:choose>
  </xsl:function>
```